src/backend/distributed/executor/multi_utility.c

```diff
@@ -883,6 +897,228 @@ ProcessAlterObjectSchemaStmt(AlterObjectSchemaStmt *alterObjectSchemaStmt,
 
 
 /*
+ * ProcessVacuumStmt processes vacuum statements that may need propagation to
+ * distributed tables. If a VACUUM or ANALYZE command references a distributed
+ * table, it is propagated to all involved nodes; otherwise, this function will
+ * immediately exit after some error checking.
+ *
+ * Unlike other Process functions within this file, this function does not
```

Comment:
Irrelevant small quibble: This isn't fully the case for all others. And probably *should* be the case for fewer (especially DDL like CREATE INDEX [CONCURRENTLY]).